src/test/kotlin/CarsTest.kt

```diff
@@ -0,0 +1,19 @@
+import domain.cars.Cars
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class CarsTest {
+    @Test
+    fun `우승자를 확인할 수 있다 `() {
+        val cars = getDummy()
+        val expected = "포비, 크론, 디노"
+
+        Assertions.assertEquals(expected, cars.getWinners())
+    }
```

Comment:
이렇게도 바로 쓸 수 있답니다! 먼저 작성해보고, 임포트 단축키를 눌러보세요!
```suggestion
        assertEquals(expected, cars.getWinners())
    }
```

Reply:
반영했습니다 ! :) 
[ffafe68](https://github.com/woowacourse/kotlin-racingcar/pull/139/commits/ffafe6802e63ec24647a471ce156d5b68f345eef)